src/main/kotlin/com/spectralogic/rioclient/RioGroups.kt

```diff
@@ -0,0 +1,41 @@
+package com.spectralogic.rioclient
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ListGroupsForRioUserResponse(
+    val rioGroupUuids: List<String>,
+    val userUuid: String,
+) : RioResponse()
+
+@Serializable
+data class CreateRioGroupRequest(
```

Comment:
There should also be a list of Domain Group UUIDs

Reply:
Let me look up what it is called. I'll get it added.